src/Sulu/Bundle/AdminBundle/Resources/js/components/Modal/index.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+import ClickModal from './ClickModal';
+import Modal from './Modal';
+
+export {ClickModal};
+export default Modal;
```

Comment:
Same as previously, I think we have to find a consistent way of dealing with this kind of stuff.